ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -238,6 +238,9 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+    CreationPolicy:
+      ResourceSignal:
+        Timeout: PT10M
```

Comment:
i think this may need 20 mins

Reply:
This value was fine during testing, but I can set it to 20.